test/test_dynamic_shapes.py

```diff
@@ -1026,6 +1026,18 @@ def test_symnode_hashing(self):
         hash(n)
         hash(m)
 
+    def test_symnode_deepcopy(self):
+        shape_env = ShapeEnv()
+
+        symnodes = (
+            create_symint(shape_env, 3),
+            create_symbool(shape_env, True),
+            create_symfloat(shape_env, 3.0),
```

Comment:
These cases don't actually go through the path that you modified below

Reply:
How can I create a symint that does go through that path?

Reply:
They don't go through the path, but we don't also care I guess? Because we only care about nested_int, which DOES go through the path.

(Though if you wanted other examples of SymInts that go throught the `torch._C._SymNode` path, you can look at `torch._C._get_constant_bool_symnode`, but those instantly evaporate upon hitting any operator and are not well tested)